### ISSUE TITLE
Vault plugin 

### ISSUE THREAD

**Issue by michaeljfarr** [JoseGamerPT/RealScoreboard#17]
Hi José,

Similar to this issue: https://github.com/JoseGamerPT/RealScoreboard/issues/7.  I've found a problem with integration with the Vault plugin that I can't resolve.

I've narrowed the problem down to Placeholders somewhere.  My guess is that in the money method RealScoreboard.vault can be 1 when RealScoreboard.Economia is still null.  So, maybe 
 - in the money method, check RealScoreboard.Economia for null rather than vault.
 - in RealScoreboard.setupEconomy, log whether it was successful in looking up the Economy provider so we can just diagnose what is going wrong with vault integration

Thanks.

**Comment by michaeljfarr** [comment]
Here is the server log file.  You can see there is a problem with Bssentials, which is probably the underlying cause of the issue.

**Comment by michaeljfarr** [comment]
I switched to EssentialsX which solved the problem. But, it would still be great if the adjustments above were made.  Possibly also a statement on the main page to say that BEssentials isn't supported (even with the API), and EssentialsX is required ... something like that would be great.

**Comment by JoseGamerPT** [comment]
Sorry but i didn´t quite understand what u said.
Could you elaborate more on your issue?

**Comment by michaeljfarr** [comment]
The behaviours are exactly the same as for issue #7 (as I linked above). It was closed for comments, so I just made a new issue and referred to it.  Check that link out because he put in screenshots showing what happens to the scoreboard.  

My experience of the error was that all the strings on the scoreboard are either turned into blank or the string "RealScoreboard".  This happens because of the SBPlayer.startScoreboard call to setPlaceHolders returns the string "RealScoreboard" whenever there is an exception.  And there is always an exception with BEssentials because the money method is throwing a null reference exception (I described that bit above as well).

The issue that I am raising is just that RealScoreboard should could produce a better log message on startup - if it couldn't find the right service provider.  This plugin is great and works well, but it was difficult to understand its prerequisites.  I had to read the code to work out that the plugin is incompatible with BEssentials - and so switching to EssentialsX was the right thing to do. 

:-) Hopefully, that's clarified things, but I can try again if it is still no good.
